go-chaos/internal/pods.go

```diff
@@ -302,6 +303,35 @@ func (c K8Client) createPortForwardUrl(names []string) *url.URL {
 	return portForwardCreateURL
 }
 
+func (c K8Client) ExecuteCommandViaDebugContainer(podName string, containerName string, debugImage string, cmd []string) error {
```

Comment:
Did you verified whether it was for real disrupting the network? Or maybe it was just silently failing? 🤔 So for example did the brokers showed that they can't connect?

Reply:
Not yet! That's one of the things I still have to do :-/

Reply:
I've tested it now with a SaaS cluster :tada: 

```
$ zbchaos --namespace 8fe648b5-1137-4d6a-af0d-be8b467fd67e-zeebe disconnect brokers --broker2NodeId 2 --broker1NodeId 1 --verbose
Flags: {1 LEADER -1  10  msg false 1 LEADER 1 2 LEADER 2 1 1713797975967 false false true false false 30 false -1 benchmark 30  8fe648b5-1137-4d6a-af0d-be8b467fd67e-zeebe 1 1 benchmark-task 0 0 0 1 -1 true}
Connecting to 8fe648b5-1137-4d6a-af0d-be8b467fd67e-zeebe
Running experiment in SaaS environment.
Patched statefulset
Port forward to zeebe-gateway-5489b4cdcf-ptfl9
Successfully created port forwarding tunnel from 46535 (local) to 26500 (remote)
Found Broker zeebe-1 with node id 1.
Found Broker zeebe-2 with node id 2.
Debug container debug-lvccl4 is running command [sh -c apt update && apt install -y iproute2 && ip route replace unreachable 10.64.73.38]
Disconnect zeebe-1 from zeebe-2
Debug container debug-658dxp is running command [sh -c apt update && apt install -y iproute2 && ip route replace unreachable 10.64.53.29]
Disconnect zeebe-2 from zeebe-1

$ zbchaos --namespace 8fe648b5-1137-4d6a-af0d-be8b467fd67e-zeebe connect brokers --verbose
Flags: {1 LEADER -1  10  msg false 1 LEADER -1 2 LEADER -1 1 1713798188959 false false true false false 30 false -1 benchmark 30  8fe648b5-1137-4d6a-af0d-be8b467fd67e-zeebe 1 1 benchmark-task 0 0 0 1 -1 true}
Connecting to 8fe648b5-1137-4d6a-af0d-be8b467fd67e-zeebe
Running experiment in SaaS environment.
Debug container debug-cv444p is running command [sh -c apt update && apt install -y iproute2 && ip route del $(ip route | grep -m 1 unreachable)]
Connected zeebe-0 again, removed unreachable routes.
Debug container debug-zg45fm is running command [sh -c apt update && apt install -y iproute2 && ip route del $(ip route | grep -m 1 unreachable)]
Connected zeebe-1 again, removed unreachable routes.
Debug container debug-tm9pdj is running command [sh -c apt update && apt install -y iproute2 && ip route del $(ip route | grep -m 1 unreachable)]
Connected zeebe-2 again, removed unreachable routes.
```

![image](https://github.com/zeebe-io/zeebe-chaos/assets/1379753/82c90109-78e5-4f59-806d-8b2b3883ca71)
![image](https://github.com/zeebe-io/zeebe-chaos/assets/1379753/e3721f30-f646-478a-902a-5d7bb0184d5c)